README.md

```diff
@@ -53,21 +58,23 @@ and a audit event is generated:
 It is possible to permanently link the executable to `libsandbox.so` without recompiling the code and avoid defining `LD_PRELOAD` environment variable. For example:
 
 Before:
+
 ```bash
 $ ldd ./helloworld
-	linux-vdso.so.1 (0x00007ffd2018e000)
-	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f8392ee9000)
-	/lib64/ld-linux-x86-64.so.2 (0x00007f83930bd000)
+    linux-vdso.so.1 (0x00007ffd2018e000)
```

Comment:
Can we drop this? This was copypasted from a terminal and `ldd` does tabs. Probably, code blocks should be excluded from linting